packages/core/src/sass/app/_common/components/cfd-poa.scss

```diff
@@ -18,7 +11,7 @@
         background-color: var(--general-main-1) !important;
     }
 
-    .dc-autocomplete {
-        margin-bottom: 0 !important;
+    &__form{
+        padding: 1.6rem 6rem;
```

Comment:
can we please have a screenshot of the design change as left/right 60px is a lot

Reply:
![Uploading Screenshot 2023-11-17 at 9.10.55 AM.png…]()

Reply:
Done. Styles are just rearranged, the initial styles that were overridden had an effect on dropdown scroll as well. Thought it was not mentioned in the bug. So fixed it with the measurements from the prod